### PR TITLE
Simplify TopLevelIncludes visitor

### PR DESCRIPTION
**Pull request by natebosch**
This is a static method on a private class so there is no usage outside
the library and it is safe to refactor.

The returned `bool` was not read, and the `messages` argument was
unused. Remove both to simplify `replace()` implementation.

### CODE REVIEW

**Review by jacob314 — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## 0.17.3
+## 0.17.3-wip
```
Comment (devoncarew) [inline CHANGELOG.md]: @natebosch - I'd like to publish this stable version of the package, as soon as we get it rolled into google3.

Comment (natebosch) [reply CHANGELOG.md]: I'd be fine either holding this PR until after the roll/publish and making this a version bump, or dropping the `-wip` and rolling to google3 with this PR included. Which do you prefer?

Comment (devoncarew) [reply CHANGELOG.md]: Either; if we land it now (w/o the version change), we can add it to the roll to google3.

Comment (natebosch) [reply CHANGELOG.md]: > if we land it now (w/o the version change)

We can do that as soon as Github starts working 😛 

```
ERROR: Permission to dart-lang/csslib.git denied to natebosch.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Pull request merged**